kitsune/wiki/views.py

```diff
@@ -1382,13 +1382,17 @@ def show_translations(request, document_slug):
     translated_locales = []
     untranslated_locales = []
 
-    translated_locales.append(document.locale)
-    translated_locales.extend(document.translations.all().values_list(
-        'locale', flat=True))
+    # Makes Sure "English (en-US)" is always on top
+    translated_locales.append((document.locale, LOCALES[document.locale].native))
+    # Gets all the translated locale codes
+    translated_locales_code = document.translations.all().values_list(
```

Comment:
`all()` here doesn't actually do anything. It is only useful when there are no other method calls after `.objects`. Please remove it.